src/renderer/styles/ControllersHub.less

```
@@ -0,0 +1,605 @@
+// ========================================
+// 控制器中心（Controllers Hub）— 像素风样式
+// 视觉语言与 DualSenseSettings.less 保持一致：
+// 像素窗锚点 + ◈ 分区标题 + 块状按钮。
+// 本文件经非 scoped <style> 引入，前缀 chub- 全局唯一。
+// ========================================
+
+@font-face {
+  font-family: 'FusionPixel';
```

Stylelint (17.14.0)

[error] Expected no quotes around "FusionPixel" (font-family-name-quotes)

(font-family-name-quotes)